NEWS.md

```diff
@@ -1,8 +1,15 @@
+# easystats 0.4.0
+
+* Includes a new package for automated reporting of statistical analyses:
+  `report`.
+
+* Moves `rvest`, `xml2`, and `remotes` from `Imports` to `Suggests`.
+
 # easystats 0.3.0
 
-* Revised startup-messages, so these can be suppressed by using 
+* Revised startup-messages, so these can be suppressed by using
   `suppressPackageStartupMessages()`.
-  
-* The CRAN-check for errors or warnings in easystats-packages is no
-  longer shown by default on startup. Use `easystats::CRAN_checks()`
-  to see them.
\ No newline at end of file
+
+* The CRAN-check for errors or warnings in `easystats`-packages is no longer
```

Comment:
These are the sorts of reformatting changes that create messy commit history that can be avoided using one-sentence-per-line.